internal/handshake/crypto_setup_test.go

```diff
@@ -418,11 +418,13 @@ var _ = Describe("Crypto Setup TLS", func() {
 					close(receivedSessionTicket)
 				})
 				clientConf.ClientSessionCache = csc
+				const serverRTT = 25 * time.Millisecond // RTT as measured by the server. Should be restored.
```

Comment:
There's no check that this is actually restored, is there?

Reply:
There was a check of that, but I removed it because the RTT is only restored for go 1.21. Any advice for this test?

Reply:
```go
!strings.Contains(runtime.Version(), "go1.20")
```